src/CoreWCF.BuildTools/src/OperationParameterInjectionGenerator.Emitter.cs

```diff
@@ -51,26 +53,28 @@ public void Decrement()
             }
 
             private readonly OperationParameterInjectionSourceGenerationContext _sourceGenerationContext;
-            private SourceGenerationSpec _generationSpec;
+            private readonly SourceGenerationSpec _generationSpec;
 
-            public Emitter(in OperationParameterInjectionSourceGenerationContext sourceGenerationContext, SourceGenerationSpec generationSpec)
+            public Emitter(in OperationParameterInjectionSourceGenerationContext sourceGenerationContext, in SourceGenerationSpec generationSpec)
             {
                 _sourceGenerationContext = sourceGenerationContext;
                 _generationSpec = generationSpec;
+                _builder = new StringBuilder();
             }
 
             public void Emit()
             {
                 foreach (var operationContractSpec in _generationSpec.OperationContractSpecs)
                 {
+
                     EmitOperationContract(operationContractSpec);
```

Comment:
You've added an extra blank line by mistake.

Reply:
I just dropped it.